internal/driver/glfw/device_web.go

```diff
@@ -0,0 +1,35 @@
+// +build js wasm
+
+package glfw
+
+import (
+	"regexp"
+	"syscall/js"
+
+	"fyne.io/fyne"
+)
+
+var AndroidId = regexp.MustCompile(`Android`)
+var BlackBerryID = regexp.MustCompile(`BlackBerry`)
+var iOSID = regexp.MustCompile(`iPhone|iPad|iPod`)
+var OperaID = regexp.MustCompile(`Opera Mini`)
+var WindowsMobileID = regexp.MustCompile(`IEMobile`)
+
+var navigator = js.Global().Get("navigator")
+var userAgent = navigator.Get("userAgent").String()
+var mobileCheck = AndroidId.MatchString(userAgent) ||
+	BlackBerryID.MatchString(userAgent) ||
+	iOSID.MatchString(userAgent) ||
+	OperaID.MatchString(userAgent) ||
+	WindowsMobileID.MatchString(userAgent)
+
+func (*glDevice) IsMobile() bool {
+	if mobileCheck {
+		return true
+	}
+	return false
+}
+
+func (*glDevice) SystemScaleForWindow(w fyne.Window) float32 {
+	return fyne.SettingsScaleAuto
```

Comment:
As mentioned above I think the scale calculations go in here - it's not really auto if we are going to match teh browser config,